src/validation.cpp

```diff
@@ -2577,7 +2577,7 @@ bool CChainState::ActivateBestChain(CValidationState &state, const CChainParams&
         if (GetMainSignals().CallbacksPending() > 10) {
             // Block until the validation queue drains. This should largely
             // never happen in normal operation, however may happen during
-            // reindex, causing memory blowup  if we run too far ahead.
+            // reindex, causing memory blow up if we run too far ahead.
```

Comment:
Technically this was already correct and perhaps slightly more natural, I'd remove this change IMO :)

Reply:
Good point! Fixed! :-)